views/picks.handlebars

```diff
@@ -132,7 +132,7 @@
             </tbody>
             </table>
 
-            <button type="button" id="submit-btn" class="btn btn-primary btn-lg">Submit Picks</button>
+            <button type="button" id="submit-btn" class="btn btn-lg">Submit Picks</button>
```

Comment:
Feel free to accuse me of being picky but I wanted this button to look like the rest of the buttons throughout the website lol